content/support/how_can_i_resolve_the_attributeerror_module_wandb_has_no_attribute_init.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "How can I resolve the `AttributeError: module 'wandb' has no attribute 'init'`?"
+toc_hide: true
+type: docs
+tags:
+  - crashing and hanging runs
+---
+
+This error might appear when trying to import `wandb` from a Python environment that does not have `wandb` installed, but there is a directory named `wandb` where the code is being executed. We recommend:
+
+```bash
+pip uninstall wandb && pip install wandb
```

Comment:
```suggestion
pip uninstall wandb; rm -rI wandb; pip install wandb
```
The `uninstall` is almost certain to fail if you are in this situation (`pip` returning a non-zero result), which means that the `&&` will never be `true`. This also removes an existing `wandb` directory but prompts if doing so would remove more than 3 files or would recursively delete directories.